src/utils/constants.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  EXTRA_GAS_PERCENTAGE: 0.25
+  EXTRA_GAS_PERCENTAGE: 0.25,
+  MAX_CONCURRENT_EVENTS: 50
```

Comment:
Perhaps we should run some performance tests to find which is the most efficient value for this. Maybe we can address this in a separate issue.